docs/reference/data-streams/lifecycle/index.asciidoc

```diff
@@ -0,0 +1,64 @@
+[role="xpack"]
+[[data-stream-lifecycle]]
+== Data stream lifecycle
+
+preview::[]
+
+A data stream lifecycle is the built-in mechanism data streams use to manage their lifecycle. It enables you to easily
+automate the management of your data streams according to your retention requirements. For example, you could configure
+the lifecycle to:
+
+* Ensure that data indexed in the data stream will be kept at least for the retention time you defined.
+* Ensure that data older than the retention period will be deleted automatically by {es} at a later time.
+
+To achieve that, it supports:
+
+* Automatic <<index-rollover,rollover>>, which chunks your incoming data in smaller pieces to facilitate better performance
+and backwards incompatible mapping changes.
+* Configurable retention, which allows you to configure the time period for which your data is guaranteed to be stored.
+{es} is allowed at a later time to delete data older than this time period.
+
+[discrete]
+[[data-streams-lifecycle-how-it-works]]
+=== How does it work?
+
+In intervals configured by <<data-streams-lifecycle-poll-interval,`data_streams.lifecycle.poll_interval`>>, {es} goes over
+each data stream and performs the following steps:
+
+1. Checks if the data stream has a data lifecycle configured, skipping any indices not part of a managed data stream.
+2. Rollover the write index of the data stream, if it fulfills the conditions defined by
```

Comment:
It would be best to make the verb form of the three bullets consistent: Checks/rolls over/applies, or Check/rollover/apply.

Reply:
Very sharp @tylerperk , thank you! I changed it.